blueprint-metadata/src/lib.rs

```diff
@@ -14,7 +14,7 @@ use rustdoc_types::{Crate, Id, Item, ItemEnum, Module};
 
 /// Generate `blueprint.json` to the current crate working directory next to `build.rs` file.
 pub fn generate_json() {
-    Config::builder().build().generate_json();
+    // Config::builder().build().generate_json();
```

Comment:
Does this still need to be commented out?

Reply:
No let me try to compile it again.